src/components/common/Gallery/Gallery.module.scss

```diff
@@ -0,0 +1,341 @@
+      }
+    }
+  }
+    .menu {
+      text-align: center;
+      text-transform: uppercase;
+    }
+
+    .navHeader {
+        display: flex;
+        flex-direction: row;
+        padding-left: 0;
+        margin-bottom: 0;
+        list-style: none; 
+        border-bottom: 1px solid $global-border-color;
+
+        .navItem {
+            width: 100%;
+
+            .navLink {
+            display: block;
+            padding: 0.5rem 1rem;
+            border: 1px solid $global-border-color;
+            text-align: center;
+            }
+
+            a {
+                position: relative;
+                color: $content-color;
+                display: block;
+                text-decoration: none;
+                cursor: pointer;
+                text-transform: uppercase;
+                font-weight: 600;
+
+                &.active {
+                    color: $primary;
+                    background-color: $light-grey-tertiary;
+                    &::before {
+                        content: "";
+                        position: absolute;
+                        left: 0;
+                        width: 100%;
+                        border-top: 4px solid $primary;
+                        transform: translateY(-250%);
+                    }
+                }
+            }
+        }
+    }
+    
+    .carousel {
+        border: 1px solid $global-border-color;
+
+        .mainPhoto {
+            position: relative;
+            padding: 80% 10px 0 10px;
+            z-index: -1;
+            background: {
+              color: $color-light-tertiary;
+              size: cover;
+              position: center;
+              repeat: no-repeat;
+            }
+        }
+
+        .photo {
+            position: relative;
+            padding: 6% 6% 6% 6%;
+            background: {
+              color: $color-light-tertiary;
+              size: cover;
+              position: center;
+              repeat: no-repeat;
+            }  
+        }
+
+        .activePhoto {
+            opacity: 0.5;
+        }  
+        
+        .thumbnail {
+            background-color: #ffffff;
+            justify-content: space-between;
+            border: 1px solid $global-border-color;
+            padding: 1%;
+        }
+
+        .arrowButton {
+            border: 0;
+            background-color: #000000;
+            padding: 2%;
+        }
+
+        .arrow {
+            border: solid #ffffff;
+            border-width: 0 3px 3px 0;
+            display: inline-block;
+            padding: 3px;
+            margin-left: auto;
+            margin-right: auto;
+            display: block !important;
+        }
+
+        .right {
+            transform: rotate(-45deg);
+            -webkit-transform: rotate(-45deg);
+            
+        }
+
+        .left {
+            transform: rotate(135deg);
+            -webkit-transform: rotate(135deg);
+        }
+
+        .tabButtons {
+            transform: translateY(25%);
```

Comment:
paddingiem raczej, inaczej Ci to może wylecieć poza element

Reply:
jest to gotowe